src/quality-patches/tool.md

```diff
@@ -3,6 +3,9 @@ group: tools
 title: Quality Patches Tool
 functional_areas:
```

Comment:
```suggestion
group: tools
functional_areas:
```

Reply:
This is required when migration metadata is present. The generator that creates the [changelog](https://devdocs.magento.com/migrated.html) of migrated topics uses it as a required parameter. Tests will fail without it.